.github/workflows/CI.yml

```diff
@@ -0,0 +1,82 @@
+name: CI
+# Run on tags or any pull request
+on:
+  push:
+    tags: ["*"]
+  pull_request:
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - 1
+        os:
+          - ubuntu-latest
+          - macOS-latest
+          - windows-latest
+        arch:
+          - x64
```

Comment:
Intervals should maybe work on `x86`?

Reply:
The doctests fail.  Tried it

Reply:
Maybe make an issue? We can probably just add a condition for only running the doctests on 64-bit.

Reply:
hmm so the doctests run with the regular julia tests. Since those would be loaded up on the x86 machine I don't think this is possible? https://github.com/invenia/Intervals.jl/pull/151/checks?check_run_id=1465597418 

Currently it looks like the issues are with doctests saying something will be `Int64` but they are `Int32` on the x86 build which makes sense. Would a fix be to just change those to `Int`?

Reply:
I think the repl output will still show it as `Int64` though. I'd just condition the `doctest` call on the architecture.

Reply:
oh like from within the julia `runtests.jl` file?